src/frontend/src/handler/mod.rs

```diff
@@ -60,10 +60,28 @@ pub async fn handle(
     match stmt {
         Statement::Explain {
             statement,
-            verbose,
-            trace,
-            ..
+            describe_alias: _,
+            analyze: _,
```

Comment:
As discussed, not supported options should be banned for now, instead of ignored